### DISCUSSION TITLE
A problem occurred configuring project ':react-native-reanimated'

### DISCUSSION THREAD

**Discussion by kunal-j10** [react-navigation/react-navigation#9221] [Q&A] [answered]
!!!someone please help !! i am new here
_After installing dependencies for react-navigation if am getting errors while building gradle ,
befour installing those dependencies every thing was working fine ,
so i tried to uninstall those dependencies and again everything worked fine , but
as soon as i reinstall those dependencies , i starts getting error._

those dependencies are :-
 **react-native-gesture-handler
 react-native-reanimated
 react-native-screens 
react-native-safe-area-context 
and @react-native-community/masked-view**

### my package.json is :- 
**

_{
  "name": "Demo7",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
    "@react-native-community/masked-view": "^0.1.10",
    "@react-navigation/native": "^5.8.10",
    "@react-navigation/stack": "^5.12.8",
    "react": "16.13.1",
    "react-native": "0.63.4",
    "react-native-gesture-handler": "^1.9.0",
    "react-native-reanimated": "^1.13.2",
    "react-native-safe-area-context": "^3.1.9",
    "react-native-screens": "^2.16.1",
    "react-navigation": "^4.4.3",
    "react-navigation-stack": "^2.10.2"
  },
  "devDependencies": {
    "@babel/core": "^7.8.4",
    "@babel/runtime": "^7.8.4",
    "@react-native-community/eslint-config": "^1.1.0",
    "babel-jest": "^25.1.0",
    "eslint": "^6.5.1",
    "jest": "^25.1.0",
    "metro-react-native-babel-preset": "^0.59.0",
    "react-test-renderer": "16.13.1"
  },
  "jest": {
    "preset": "react-native"
  }
}_

**

### and the error i am getting is :-
**> Configure project :react-native-reanimated
Still waiting for package manifests to be fetched remotely.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7/userguide/command_line_interface.html#sec:command_line_warnings

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\Kunal\Demo7\node_modules\react-native-reanimated\android\build.gradle' line: 89

* What went wrong:
A problem occurred configuring project ':react-native-reanimated'.
> Failed to find Build Tools revision 29.0.2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 18s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup. Run CLI with --verbose flag for more details.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081**

![error reanimated](https://user-images.githubusercontent.com/77331516/104312547-c6478600-54fc-11eb-9c34-c557cfaf996d.PNG)

**Comment by kunal-j10** [comment, marked as answer]
Well, I was able to resolve it by changing my **Build revision tool to 29.0.3**, as many packages like react-navigation, react-redux, react-firebase are not being supported in Build revision tool 30.0.x, So, it's better to use 29.0.3.

**Reply by raajnadar** [reply]
why are you using 30.0.x? I think the latest RN version supports 29.0.x

**Reply by kunal-j10** [reply]
yeah, I was making that mistake initially but now I am using 29.0.x

**Reply by raajnadar** [reply]
Just found this https://github.com/react-navigation/react-navigation/issues/8981#issuecomment-731233042

**Reply by YunusEmreNalbant** [reply]
hi guys,

My buildToolsVersion = "29.0.3" but the problem continues :(

**Reply by kunal-j10** [reply]
@YunusEmreNalbant  You might be using reanimated 2  just go through [Roadmap for v6 ](https://github.com/react-navigation/react-navigation/issues/8981#issuecomment-731233042)  as Reanimated 2 requires  Turbo Modules which aren't default in React Native yet.

**Reply by YunusEmreNalbant** [reply]
@kunal-j10 
Thank you for your answer, brother. Yes, I was using v2. But now I have installed v1.13.1, but it gives the following error:
![Screenshot_2](https://user-images.githubusercontent.com/29780061/110831831-ae497400-82ab-11eb-81fc-063889f75bc3.png)

**Reply by raajnadar** [reply]
This thread will answer your question https://github.com/software-mansion/react-native-reanimated/issues/1798#issuecomment-796914997

**Reply by YunusEmreNalbant** [reply]
thank you for help.. FINALLY WORKED !!!

**Comment by giladm** [comment]
I am using react-native 0.69.3 and also had this issue using reanimated@2.10. I then updated to @2.11.0 and it fixed my compile error.

**Comment by ggrachdev** [comment]
I tried setting - "react-native-reanimated": "2.11.0" and it didn't work. As a result, the problem was solved by installing the latest version of "react-native-screens" and managed to build the apk through android studio

**Reply by alexander0205** [reply]
That fix my error: [https://stackoverflow.com/a/47365920/5963210](https://stackoverflow.com/a/47365920/5963210)

**Reply by PhillipMwaniki** [reply]
Thanks, this worked perfectly.
